src/Symfony/Component/Security/Acl/Domain/Acl.php

```diff
@@ -511,7 +511,7 @@ private function insertAce($property, $index, $mask, SecurityIdentityInterface $
      */
     private function insertFieldAce($property, $index, $field, $mask, SecurityIdentityInterface $sid, $granting, $strategy = null)
     {
-        if (0 === strlen($field)) {
+        if (empty($field)) {
```

Comment:
according to phpdoc field must be a string. so no need to support something else. also empty is wrong since it fails with "0" as kalessil said.